index.html

```diff
@@ -0,0 +1,105 @@
+                <figure>
+                    <img src="../img/1.jpg" alt="steps order">
+                    <figcaption>The ten Cuisenaire rods arranged in order as a staircase with step size White.</figcaption>
+                </figure>
+            </div>
+            <div>
+                <p>equations using plus, minus, multiply, and fractions as operators, and translate these into arithmetic. The results had been impressive throughout key stage 1. In Year 2 we wanted to take it a step further by running computing lessons using the same use of natural language and level of sophistication.</p>
+                <h2>Why Haskell?</h2>
+                <p>Haskell is closely aligned to conceptual mathematics and number theory, the inspiration for Gattegno Mathematics. It has four attributes that make it particularly appropriate for mathematics education:</p>
+                <ul>
+                    <li>Its syntax is very close to traditional mathematical notation (but beware that it can be too terse).</li>
+                    <li>Variables are statically typed, which means that programmers are protected by the compiler from passing invalid information as an argument (input) or output value.</li>
+                    <li>The Haskell interpreter evaluates functions ‘lazily’, only when required. This makes it possible to program infinite data structures in finite memory.</li>
+                    <li>Information is passed from the arguments to functions by pattern matching. This is an intuitive way for programmers to envisage and reason about the behaviour of their programs.</li>
+                </ul>
+            </div>
+            <div>
+                <figure>
+                    <img src="../img/page1_1.jpg" alt="math">
+                    <figcaption>Equivalent expressions using Gattegno’s rod codes to represent the relationship between a rod and equivalent trains of one colour</figcaption>
+                </figure>
+                <h2>Starting with a staircase</h2>
+                <p>To start, we went back to a familiar pattern that Gattegno introduces in the textbooks, called a staircase. This is built from one rod of each colour arranged in size order (figure 1). The children were able to physically build the staircase. However, because they were so familiar with the pattern, they could also access it virtually, which would help with debugging and logical reasoning later. A role-play game to establish computational thinking was introduced before any actual programming began. This allowed the children to familiarise themselves with the function calls <strong>successor</strong> and <strong>predecessor (succ, pred)</strong> while traversing the staircase. The children would take turns to ask each other questions such as <strong>succ White</strong> (answer Red) or <strong>pred Orange</strong> (answer Blue). To help the children access the new programming vocabulary, one child</p>

```

Comment:
It's ok to use `strong` for such cases, but taking into account that in all other parts of the text you selected this different font using `span` it looks strange that you chose `strong` for it in this paragraph. Consistency is one of the principles we try to stick to, so I would use spans here instead